sklearn/preprocessing/label.py

```diff
@@ -352,8 +352,7 @@ def fit_transform(self, y):
         Y : numpy array or CSR matrix of shape [n_samples, n_classes]
             Shape will be [n_samples, 1] for binary problems.
         """
-        self.fit(y)
-        return self.transform(y)
+        return super(fit_transform, self).fit_transform(y)
```

Comment:
I made a mistake here, it should be `return super(LabelBinarizer, self).fit_transform(y)` . I'm going to wait for feedback before committing the change to avoid polluting the commit history in case of additional issues.